skbase/lookup/_lookup.py

```diff
@@ -624,127 +707,202 @@ def get_package_metadata(
 def all_objects(
     object_types=None,
     filter_tags=None,
+    exclude_objects=None,
     exclude_estimators=None,
```

Comment:
same as above - do we need deprec?

Reply:
This is more of a question for you and sktime.


If you are planning to wrap skbase's function in a sktime one,  you could handle deprecation there.

My thoughts are let's not deprecate the new functions added in skbase (e.g., get_package_metadata). I'm fine not deprecating the old ones too, but had been due to potential difference relative to sktime.

Reply:
happy without deprecation - I know what's coming, after all, and there is no hard dep which could break it for users

Reply:
Also removed this deprecation.